tests/test_grounding.py

```diff
@@ -51,7 +56,12 @@ def test_terminal_grounder(self):
         def mock_input(prompt):
             return next(mock_input_gen)
 
-        with mock.patch('__builtin__.raw_input', mock_input):
```

Comment:
I believe you can mock `six.input` here? I'm not sure, but it's worth trying, it's much less code to maintain